src/main/resources/data/minecolonies/tags/items/mechanic_product.json

```diff
@@ -1,31 +1,31 @@
-{
-  "replace": false,
-  "values": [
-    "minecraft:blue_ice",
-    "minecraft:enchanting_table",
-    "minecraft:torch",
-    "minecraft:soul_torch",
-    "minecraft:jack_o_lantern",
-    "minecraft:lantern",
-    "minecraft:soul_lantern",
-    "minecraft:packed_ice",
-    "minecraft:tripwire_hook",
-    "#minecraft:rails",
-    "#forge:storage_blocks",
-    "minecraft:bone_block",
-    "minecraft:sea_lantern",
-    "minecraft:stone_pressure_plate",
-    "minecraft:polished_blackstone_pressure_plate",
-    "#minecraft:wooden_pressure_plates",
-    "minecraft:light_weighted_pressure_plate",
-    "minecraft:heavy_weighted_pressure_plate",
-    "minecraft:sticky_piston",
-    "minecraft:trapped_chest",
-    "minecraft:ender_chest",
-    "minecraft:lever",
-    "minecraft:daylight_detector",
-    "minecraft:dried_kelp_block",
-    "minecraft:honey_block",
-    "minecraft:honeycomb_block"
-  ]
-}
+{
+  "replace": false,
+  "values": [
+    "minecraft:blue_ice",
+    "minecraft:enchanting_table",
+    "minecraft:torch",
+    "minecraft:soul_torch",
+    "minecraft:jack_o_lantern",
+    "minecraft:lantern",
+    "minecraft:soul_lantern",
+    "minecraft:packed_ice",
+    "minecraft:tripwire_hook",
+    "#minecraft:rails",
+    "#forge:storage_blocks",
+    "minecraft:bone_block",
+    "minecraft:sea_lantern",
+    "minecraft:stone_pressure_plate",
+    "minecraft:polished_blackstone_pressure_plate",
+    "#minecraft:wooden_pressure_plates",
+    "minecraft:light_weighted_pressure_plate",
+    "minecraft:heavy_weighted_pressure_plate",
+    "minecraft:sticky_piston",
+    "minecraft:trapped_chest",
+    "minecraft:ender_chest",
+    "minecraft:lever",
+    "minecraft:daylight_detector",
+    "minecraft:dried_kelp_block",
+    "minecraft:honey_block",
+    "minecraft:honeycomb_block"

```

Comment:
This file still has changes?

Reply:
not sure what's going on with that, looks like that's the case with blacksmith_products too despite there being no actual changes. Maybe it's because I switched from my laptop to desktop halfway through?

Reply:
could be line endings, could be tabs/spaces auto-reformatting, could be something else. if you're using gitkraken I know a way to revert the changes

Reply:
Just started using gitkraken recently, if you could let me know that would be great! Maybe message me on Discord if you'd like to avoid cluttering up the comments? Brick_man#0096

Reply:
If you right-click on the commit there's an option to "revert" it, you can choose to only revert certain files/etc

Reply:
thanks!